atc/db/pipeline_factory.go

```diff
@@ -26,35 +26,21 @@ func NewPipelineFactory(conn Conn, lockFactory lock.LockFactory) PipelineFactory
 
 func (f *pipelineFactory) VisiblePipelines(teamNames []string) ([]Pipeline, error) {
 	rows, err := pipelinesQuery.
-		Where(sq.Eq{"t.name": teamNames}).
-		OrderBy("team_id ASC", "ordering ASC").
+		Where(sq.Or{
+			sq.Eq{"t.name": teamNames},
+			sq.And{
+				sq.NotEq{"t.name": teamNames},
```

Comment:
I think we can have it only check that it is `public: true` and remove the `t.name != teamNames`. 🤔

Reply:
Removed that. Good call! Thx.